src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/DataReaderTest/DataReaderTest.cs

```diff
@@ -145,6 +146,60 @@ public static void CheckSparseColumnBit()
             }
         }
 
+        [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup), nameof(DataTestUtility.IsNotAzureServer))]
```

Comment:
We need a comprehensive test to cover every collation. It can be new work to do.
Any thoughts @cheenamalhotra?

Reply:
Yes, it's backlogged.